snow/engine/avalanche/bootstrap/bootstrapper.go

```diff
@@ -125,84 +129,76 @@ func (b *bootstrapper) Clear(context.Context) error {
 // response to a GetAncestors message to [nodeID] with request ID [requestID].
 // Expects vtxs[0] to be the vertex requested in the corresponding GetAncestors.
 func (b *bootstrapper) Ancestors(ctx context.Context, nodeID ids.NodeID, requestID uint32, vtxs [][]byte) error {
```

Comment:
This function was refactored to be more similar to the snowman code. The major difference here is that we no longer "optimistically" accept any vertices. If we didn't request it, we will drop it. (which should never happen anyways because the message should have been dropped at the network level now)